lib/routes.js

```diff
@@ -2,6 +2,8 @@
 
 var errors = require('./errors')
 var extension = require('./extension')
+const logger = require('winston')
+const { stopServer } = require('./server')
```

Comment:
server requires this routes file again. So, not sure if we can follow this pattern